plugin-repos/ufc-scoreboard/base_odds_manager.py

```
@@ -0,0 +1,284 @@
+
+        Returns:
+            Dictionary containing odds data or None if unavailable
+        """
+        if sport is None or league is None or event_id is None:
+            raise ValueError("Sport, League, and event_id cannot be None")
+
+        if comp_id is None:
+            comp_id = event_id
+
+        # Use provided interval or default
+        interval = update_interval_seconds or self.update_interval
```

Ruff (0.15.0)

[error] Local variable `interval` is assigned to but never used

Remove assignment to unused variable `interval`

(F841)